packages/aws-cdk/test/integ/cli/test-cdk-ssm-parameter-provider-error.sh

```diff
@@ -15,7 +15,8 @@ function cdk_synth() {
 }
 
 assert "cdk_synth ${STACK_NAME_PREFIX}-missing-ssm-parameter -c test:ssm-parameter-name=${parameterName}" <<HERE
-SSM parameter not available in account ${account}, region ${region}: ${parameterName}
+[Error at /cdk-toolkit-integration-missing-ssm-parameter] SSM parameter not available in account 993655754359, region us-east-1: /does/not/exist
```

Comment:
Are you sure we don’t want to use substitution here?

Reply:
+1

does this work:
```
[Error at ${STACK_NAME_PREFIX}-missing-ssm-parameter] SSM parameter not available in account ${account}, region ${region}: ${parameterName}
```

or adding `errorKey=[Error at /cdk-toolkit-integration-missing-ssm-parameter]` and use it as `${errorKey}`  ahead of the rest of the existing error message?

Reply:
Doh, you are correct.